### ISSUE TITLE
Missing en-core-web-sm for conda environment ?

### ISSUE THREAD

**Issue by damaskino** [MLforHealth/MIMIC_Extract#28]
Hi there,

I've tried following the README, but I'm getting an error at Step 2 where I need to create the conda environmen. It looks like en-core-web-sm is either missing or the specified version is not available? 
Looking up the name on pypi.org didn't yield the name as a result.

```
conda env create --force -f ../mimic_extract_env_py36.yml

Collecting package metadata (repodata.json): done
Solving environment: done
Preparing transaction: done
Verifying transaction: done
Executing transaction: \ Enabling notebook extension jupyter-js-widgets/extension...
      - Validating: OK

done
Installing pip dependencies: / Ran pip subprocess with arguments:
['/home/chen/miniconda3/envs/mimic_extract_py36/bin/python', '-m', 'pip', 'install', '-U', '-r', '/home/chen/MIMIC_Extract/condaenv._xem_m0y.requirements.txt']
Pip subprocess output:
Collecting blis==0.4.1 (from -r /home/chen/MIMIC_Extract/condaenv._xem_m0y.requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/41/19/f95c75562d18eb27219df3a3590b911e78d131b68466ad79fdf5847eaac4/blis-0.4.1-cp36-cp36m-manylinux1_x86_64.whl
Collecting catalogue==1.0.0 (from -r /home/chen/MIMIC_Extract/condaenv._xem_m0y.requirements.txt (line 2))
  Using cached https://files.pythonhosted.org/packages/6c/f9/9a5658e2f56932e41eb264941f9a2cb7f3ce41a80cb36b2af6ab78e2f8af/catalogue-1.0.0-py2.py3-none-any.whl
Collecting en-core-web-sm==2.1.0 (from -r /home/chen/MIMIC_Extract/condaenv._xem_m0y.requirements.txt (line 3))

Pip subprocess error:
  ERROR: Could not find a version that satisfies the requirement en-core-web-sm==2.1.0 (from -r /home/chen/MIMIC_Extract/condaenv._xem_m0y.requirements.txt (line 3)) (from versions: none)
ERROR: No matching distribution found for en-core-web-sm==2.1.0 (from -r /home/chen/MIMIC_Extract/condaenv._xem_m0y.requirements.txt (line 3))

failed

CondaEnvException: Pip failed
```

**Comment by ducnx** [comment]
You have to replace the dependency `en-core-web-sm==2.1.0` by the direct URL to the package: `https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.1.0/en_core_web_sm-2.1.0.tar.gz`.
It works for me.

**Comment by damaskino** [comment]
That did it for me, replaced the line in the .yml and now the environment is succesfully created thanks alot!

**Comment by mmcdermott** [comment]
Oh, thanks @ducnx! We'll update the yml file with that fix in general, so nobody else is hit by this too.

**Comment by ducnx** [comment]
You're welcome. I think the alternate instructions ("no Makefile" version) would also work, as we download `en_core_web_sm` via spacy.

There's another minor error in the .yml file though, the environment name is specified as `mimic_extract_py36` but the instruction says `conda activate mimic_data_extraction`.

**Comment by mmcdermott** [comment]
Pushed this fix in #32
